builtin/client/death_formspec.lua

```diff
@@ -2,9 +2,9 @@
 -- handled by the engine.
 
 core.register_on_death(function()
-	core.display_chat_message("You died.")
+	core.display_chat_message(core.gettext("You died."))
 	local formspec = "size[11,5.5]bgcolor[#320000b4;true]" ..
-		"label[4.85,1.35;" .. fgettext("You died") ..
```

Comment:
why not store the core.gettext("You died.") in a single variable to re-use, it permits to prevent double core call to gettext

Reply:
After all, why not `fgettext`?